SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,201 @@
+        
+        feedbackData = [feedback]
+    }
+    
+    func setupNavigationBar() {
+        navigationController?.navigationBar.barTintColor = UIColor.navigationTintColor
+        let attributes = [
+            NSAttributedString.Key.font: UIFont._21RobotoMedium!,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedTitle = NSAttributedString(string: "Feedback", attributes: attributes)
+        title = attributedTitle.string
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            image: UIImage(named: "pen"),
+            style: UIBarButtonItem.Style.plain,
+            target: self,
+            action: #selector(handleNavigationBarRightTap)
+        )
+    }
+    
+    func setupFeedbackLabel() {
+        feedbackLabel.lineBreakMode = .byWordWrapping
+        feedbackLabel.numberOfLines = 0
+
+        // Format the alignment and spacing between lines
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineSpacing = 10
+        
+        let title = "No Feedback Yet\n"
+        let subtitle = "See feedback conversations here"
+        
+        let titleAttributes: [NSAttributedString.Key : Any] = [
+            NSAttributedString.Key.font: UIFont._17RobotoMedium!,
+            .paragraphStyle: paragraphStyle,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+        
+        let subtitleAttributes = [
+            NSAttributedString.Key.font: UIFont._13RobotoRegular!,
+            NSAttributedString.Key.foregroundColor: UIColor.subtitleColor
+        ]
+        attributedText.append(NSAttributedString(string: subtitle, attributes: subtitleAttributes))
+
+        feedbackLabel.attributedText = attributedText
+        feedbackLabel.sizeToFit()
+        view.addSubview(feedbackLabel)
+    }
+    
+    func setupConversationButton() {
+        newConversationButton.setTitle("Start Conversation", for: .normal)
+        newConversationButton.backgroundColor = UIColor.themeColor
+        newConversationButton.titleLabel?.font = UIFont._17RobotoMedium
+        newConversationButton.layer.cornerRadius = 22
+        view.addSubview(newConversationButton)
+    }
+    
+    func setupFeedbackTableView() {
+        feedbackTableView.delegate = self
+        feedbackTableView.dataSource = self
+        feedbackTableView.tableFooterView = UIView()
+        feedbackTableView.register(FeedbackTableViewCell.self, forCellReuseIdentifier: FeedbackTableViewCell.reuseID)
+        view.addSubview(feedbackTableView)
+    }
+    
+    func setupConstraints() {
+        if !feedbackData.isEmpty {
+            feedbackTableView.snp.makeConstraints { make  in
+                make.leading.trailing.top.bottom.equalToSuperview()
+            }
+        } else {
```

Comment:
So we added the DZN empty stuff pod to help handle this but we haven't used it. If u want a reference on a good way to use it, peep [transit code](https://github.com/cuappdev/ithaca-transit-ios/blob/master/TCAT/Controllers/RouteOptionsViewController%2BExtensions.swift#L320)